filebeat/input/log/harvester.go

```diff
@@ -318,21 +325,21 @@ func (h *Harvester) Run() error {
 		if err != nil {
 			switch err {
 			case ErrFileTruncate:
-				logp.Info("File was truncated. Begin reading file from offset 0: %s", h.state.Source)
+				logger.Infof("File was truncated. Begin reading file from offset 0: %s", h.state.Source)
```

Comment:
Nit: as the path is now part of the log, there is no need for adding `h.state.Source` to the log line.

Reply:
True, I'm kinda lazy here, as I didn't want to break all integration tests (some are already broken due to changes in the log output).

Plus in Logs UI or Kibana discover, one might still look for the full message in the first place. Here the additional fields mostly provide filtering capabilities.